PolicyServicePkg/PolicyService/DxeMm/PolicyCommon.c

```diff
@@ -10,7 +10,15 @@
 #include "PolicyCommon.h"
 #include "PolicyInterface.h"
 
-STATIC LIST_ENTRY  mPolicyListHead = INITIALIZE_LIST_HEAD_VARIABLE (mPolicyListHead);
+LIST_ENTRY  mPolicyListHead = INITIALIZE_LIST_HEAD_VARIABLE (mPolicyListHead);
+LIST_ENTRY  mNotifyListHead = INITIALIZE_LIST_HEAD_VARIABLE (mNotifyListHead);

```

Comment:
Did we need to drop the STATIC?

Reply:
Yeah, for the unittest to be able to manipulate the list to reset and check state

Reply:
That's where I just have the unit test include the c file so it can use the statics directly, which is not a great pattern, but preserves the static :). No need to change here, though.